lms/djangoapps/commerce/urls.py

```diff
@@ -4,10 +4,12 @@
 
 from django.conf.urls import patterns, url
 
-from .views import OrdersView, checkout_cancel
+from .views import BasketsView, checkout_cancel
 
 urlpatterns = patterns(
     '',
-    url(r'^orders/$', OrdersView.as_view(), name="orders"),
+    url(r'^baskets/$', BasketsView.as_view(), name="baskets"),
     url(r'^checkout/cancel/$', checkout_cancel, name="checkout_cancel"),
+    # For backwards compatibility with js clients during intial release
```

Comment:
Including a unique identifier (e.g., XCOM-214) in a comment near this and other changes introduced for backwards compatibility would make the clean-up process easier.